src/main/java/JerChatBot.java

```diff
@@ -0,0 +1,177 @@
+        System.out.println("~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~");
+        System.out.println(" Please tell me what to do next");
+        System.out.println("~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~");
+    }
+
+    private static void printExitMessage() {
+        System.out.println("~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~");
+        System.out.println(" End of Program");
+        System.out.println("~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~");
+    }
+
+    private static void printTaskList() {
+        System.out.println("~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~");
+        System.out.println(" Current Task List:");
+        if (taskCount == 0) {
+            System.out.println(" No tasks in your list currently.");
+        } else {
+            for (int i = 0; i < taskCount; i++) {
+                System.out.println((i + 1) + ". " + tasks[i]);
+            }
+        }
+        System.out.println("~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~");
+    }
+
+    private static void handleMarkCommand(String[] commands) {
+        if (commands.length < 2) {
+            printInvalidTaskMessage();
+            return;
+        }
+
+        try {
+            int taskIndex = Integer.parseInt(commands[1]) - 1;
+            if (isValidTaskIndex(taskIndex)) {
+                tasks[taskIndex].markAsDone();
+                System.out.println("~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~");
+                System.out.println(" Marked this task as done:");
+                System.out.println(" " + tasks[taskIndex]);
+                System.out.println("~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~");
+            } else {
+                printInvalidTaskMessage();
+            }
+        } catch (NumberFormatException e) {
+            printInvalidTaskMessage();
+        }
+    }
+
+    private static void handleUnmarkCommand(String[] commands) {
+        if (commands.length < 2) {
+            printInvalidTaskMessage();
+            return;
+        }
+
+        try {
+            int taskIndex = Integer.parseInt(commands[1]) - 1;
+            if (isValidTaskIndex(taskIndex)) {
+                tasks[taskIndex].removeAsDone();
+                System.out.println("~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~");
+                System.out.println(" Marked this task as not done:");
+                System.out.println(" " + tasks[taskIndex]);
+                System.out.println("~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~");
+            } else {
+                printInvalidTaskMessage();
+            }
+        } catch (NumberFormatException e) {
+            printInvalidTaskMessage();
+        }
+    }
+
+    private static void printInvalidTaskMessage() {
+        System.out.println("~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~");
+        System.out.println(" Invalid command or task number. Please try again.");
+        System.out.println("~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~");
+    }
+
+    private static void addTask(String taskDescription) {
```

Comment:
maybe add some comments to explain the function of the method and its parameters